### PR TITLE
Split test result topic view into a separate tab.

### PR DESCRIPTION
**Pull request by frgtn**
Also makes the topic tab the default. See result in the screenshot below (don't mind the missing images, that's a problem with my local setup):

screen shot 2016-10-01 at 15 00 19

This is rough PR as some the code could use some cleanup (like removing unused props after splitting fraction result tab into two).

### CODE REVIEW

**Comment by frgtn** [comment]
Please review but don't merge this yet as this change is still being discussed. I don't know how approvals work.

File: manoseimas/compatibility_test/client/app/ResultsView/SimilarityTopics/index.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      fractions: PropTypes.array,
+      mps: PropTypes.array,
+      topics: PropTypes.array,
+      saveAnswer: PropTypes.func,
+      getResults: PropTypes.func,
+      toggleImportance: PropTypes.func,
+      show_header: PropTypes.bool,
+      showHeader: PropTypes.func,
+      setTab: PropTypes.func,
+      hideHeader: PropTypes.func,
```
Comment (chompomonim) [inline manoseimas/compatibility_test/client/app/ResultsView/SimilarityTopics/index.jsx]: Why do you add this prop here if you don't use it?
Same question to a few other props here.

Comment (frgtn) [inline manoseimas/compatibility_test/client/app/ResultsView/SimilarityTopics/index.jsx]: Thanks for taking a look!

What I did was just splitting SimilarityTopicsContainer out from SimilarityFractionsContainer so these props got left over. These have been cleaned up.

**Comment by chompomonim** [comment]
Good job! You can now merge it at any time.

**Pull request merged**